packages/maven/sentry-apache-http-client-5/4.1.0.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "io.sentry:sentry-apache-http-client-5",
+    "canonical": "maven:io.sentry:sentry-apache-http-client-5",
+    "version": "4.1.0",
+    "package_url": "https://search.maven.org/artifact/io.sentry/sentry-apache-http-client-5",
+    "repo_url": "https://github.com/getsentry/sentry-java",
+    "main_docs_url": "https://docs.sentry.io/platforms/java/"
```

Comment:
Unless we get a dedicated page, we just point to the main Java docs.

Today it's under `/platform/java/transport` but I'm not sure that'll be a stable link

Reply:
lets keep the java root page, agreed